redoing transaction updates for serverless in a new branch (#2416)
getsentry/sentry-docs: src/platforms/python/guides/aws-lambda/index.mdx

The timeout warning is sent only if the timeout in the Lambda Function configuration is set to a value greater than one second.

## Behavior

With the AWS Lambda integration enabled, the Python SDK will:

> - Automatically report all events from your Lambda Functions
> - You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink>

Reviewer: ```suggestion
- Automatically report all events from your Lambda Functions. You can [modify the transaction sample rate] <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink> using `traces_sample_rate`.
```
